chingu-x/chingu-connect
# New/#74 create connection mutation form

Closes #74 

## Purpose
* Create a form for new connections

## Changes/Approach
* Merged `dev` branch to sync @Bigghead 's code
* Created a `mutations` directory on the FE
* Created a graphql `CREATE_CONNECTION` mutation on the FE
* Created a `new connection` component with a form
* Connected apollo graphql mutation with form component

## Still need to do:
* Sort connections
* Show expiration time

## Testing
* Download PR to local device
* Check that helpboard component shows all connections
* Click `new connection` button to see form component
* Create a new form
* Make sure you're re-directed back to helpboard copmonent
* Check that your newly created connection shows
